liblouis/compileTranslationTable.c

```diff
@@ -5251,6 +5255,17 @@ lou_free(void) {
 		}
 		translationTableChain = NULL;
 	}
+	if (displayTableChain != NULL) {
+		DisplayTableChainEntry *currentEntry = displayTableChain;
+		DisplayTableChainEntry *previousEntry;
+		while (currentEntry) {
+			freeDisplayTable(currentEntry->table);
```

Comment:
what happens if the display table is NULL?

Reply:
If `currentEntry` is `NULL` you would get undefined behavior. But that will never happen.

Reply:
OK